src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,46 @@
+.stars {
+  width: 166px;
+  height: 16px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 16px 16px 0;
```

Comment:
The padding property is set to '16px 16px 0', which means top and bottom paddings are different. This might be intentional, but ensure it aligns with the design requirements.